platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/InputValidation.java

```diff
@@ -31,14 +31,22 @@
     private static final String DEFAULT_FILE = "file" + DEFAULT_EXTENSION;
 
     private static final List<String> BAD_FILES = Arrays.asList(System.getProperty("bad.files")
+            .toLowerCase()
```

Comment:
❓ Was there an issue with some file names not being all lowercase? Why does this need to happen?